examples/complete/root-modules/network/stacks/dev.yaml

```diff
@@ -1,3 +1,4 @@
+kind: StackConfigV1
```

Comment:
It's going to conflict with the PR https://github.com/masterpointio/terraform-spacelift-automation/pull/42, isn't it?

Reply:
Will fix it in post!